test/redbird_test.exs

```diff
@@ -57,6 +57,29 @@ defmodule RedbirdTest do
       assert get_session(conn, :foo) == "bar"
     end
 
+    test "it reuses the session key between requests" do
```

Comment:
Thank you for adding this test! 🎉  It was helpful for me to be able to run it locally to understand the failure you're talking about. 👍